omaha/common/omaha_customization_unittest.cc

```diff
@@ -126,18 +126,22 @@ TEST(OmahaCustomizationTest, Constants_Certificate) {
 }
 
 TEST(OmahaCustomizationTest, Constants_OmahaAppId_String) {
-  EXPECT_STREQ(_T("{2070893A-B7CF-42FD-9BA1-F00E04A9D766}"), GOOPDATE_APP_ID);
-  EXPECT_STREQ(_T("{2070893A-B7CF-42FD-9BA1-F00E04A9D766}"),
+#ifdef GOOGLE_UPDATE_BUILD
```

Comment:
This is a pretty useless test; rolled it back and enclosed it in GOOGLE_UPDATE_BUILD, which I am upstreaming